code/modules/mob/living/carbon/diona_base.dm

```diff
@@ -81,19 +81,19 @@ var/list/diona_banned_languages = list(
 	//But diona gestalts gain nutrition by extracting matter from gases in the air. If a gestalt spends a long time in space or on the asteroid, it may need to actually eat food
 	//For simplicity, we'll assume any gas is fine, so they'll just absorb nutrition based on pressure
 	if (!pressure)
-		return
+		return 0
 
 	if (DS.nutrient_organ)
 		if (DS.nutrient_organ.is_broken())
-			return
+			return 0
```

Comment:
because the output will be fed directly into another function. I guess it wouldn't be an issue, but its good practise.

I try to return a number wherever possible (1/0 for success/fail) even if i have no immediate use for that return. It gives something to hook into for future expansion, which is something ive had to add to hundreds of functions that lacked the feedback